frontend/src/components/repartitions/Nouveau.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import Container from "react-bootstrap/Container";
+import { authService } from "../../App";
+import { RepSidebar } from "./RepSidebar";
+import { PageTitle } from "../utils/PageTitle";
+
+export const Nouveau =({
```

Comment:
English

Reply:
+ Use a more specific name, like `NewRepartition` or `CreateRepartition`. Otherwise, it clutters the global namespace